prov/efa/src/efa_fabric.c

```diff
@@ -710,14 +710,15 @@ static int efa_alloc_info(struct efa_device *device, struct fi_info **info,
 	if (ret)
 		goto err_free_info;
 
-	name_len = strlen(EFA_FABRIC_PREFIX) + INET6_ADDRSTRLEN;
+	name_len = 3;
```

Comment:
Instead of using a number, Maybe define a constant `fabbric_name` and apply `strlen` on it?

Reply:
Sure